app/modules/sightings/models.py

```diff
@@ -186,6 +186,8 @@ def check_jobs(cls):
 
     def check_all_job_status(self):
         jobs = self.jobs
+        if not jobs:
```

Comment:
This change also needs to go into the check_all_job_status in asset group sighting but I can make that change if you like

Reply:
can do.